pkg/workflows/testdata/fixtures/workflows/workflow_schema.json

```diff
@@ -64,6 +64,15 @@
     },
     "workflowSpecYaml": {
       "properties": {
+        "name": {
+          "type": "string",
+          "maxLength": 10,
+          "minLength": 10
```

Comment:
Could this not be less than 10?

Reply:
how should this be encoded for the contract? the contract forces exactly ten bytes.

i agree that this is pretty gross, but i'm concerned something will go wrong in the serialization.

Reply:
we decided to add padding. see aggregator.Metadata.padWorkflowName